pyvista/plotting/widgets.py

```diff
@@ -140,6 +140,7 @@ def add_mesh_clip_box(
         widget_color=None,
         outline_translation=True,
         merge_points=True,
+        crinkle=False,
```

Comment:
As Codecov says, it seems to need to be tested. I think it should be just to run it.

Reply:
Resolved in e4c9a2ac2b86ffc57822a06df9a3251eb69c639f